core/domain/auth_jobs_one_off.py

```diff
@@ -0,0 +1,195 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Auth-related one-off jobs."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import ast
+import itertools
+
+from core import jobs
+from core.domain import auth_domain
+from core.platform import models
+from core.platform.auth import firebase_auth_services
+import python_utils
+
+import firebase_admin
+from firebase_admin import auth as firebase_auth
+from firebase_admin import exceptions as firebase_exceptions
+
+FIREBASE_IMPORT_USERS_MAX_LEN = 1000
```

Comment:
Not sure what "len" means here, maybe MAX_USERS_THAT_FIREBASE_CAN_IMPORT_AT_ONCE or something with those semantics.

Reply:
How's this?